assembl/static2/package.json

```diff
@@ -66,6 +66,7 @@
     "json-loader": "^0.5.4",
     "lodash": "^4.17.4",
     "moment": "^2.18.1",
+    "piwik-react-router": "assembl/piwik-react-router#skip_piwik_initialization",
```

Comment:
Oh okay, I did not know that we wanted to make our code support several analytics tracking services. Is it really? I've only heard of piwik support until now. Are you sure it is part of the requirements, or are we going scope creep? (in local.ini, the variable is called `web_analytics_piwik_script`, there is no generic `web_analytics_script`)

Do you have an idea how we could plug a generic tracking system to listen to changes in URL which are made by the React application? I have the intuition that it would imply a much more complicated architecture, and re-implementing big parts of what this library does and which is really tight to React router (so using an external lib which is maintained helps us for this, in case there are future changes in how react router does things which would create a need for changing this code). All this for no real added value at the moment, as I don't see on our roadmap anything about other tracking tools.